packages/atlas/src/components/_overlays/ContextMenu/ContextMenu.tsx

```diff
@@ -19,7 +19,7 @@ type ContextMenuProps = { items: MenuItemProps[] } & Omit<PopoverProps, 'content
 export const ContextMenu: React.FC<ContextMenuProps> = ({ children, items, ...rest }) => {
   const contextMenuInstanceRef = useRef<PopoverImperativeHandle>(null)
   return (
-    <Popover hideOnClick ref={contextMenuInstanceRef} {...rest}>
+    <Popover hideOnClick ref={contextMenuInstanceRef} {...rest} placement="left">
```

Comment:
Why change the placement for all contextMenus instead of passing it as a prop here?

Reply:
Oh was testing placement and forgot to remove it